src/test/java/com/google/api/codegen/testdata/ruby/ruby_library.baseline

```diff
@@ -1259,12 +1269,13 @@ module Library
 
       def create_shelf \
           shelf,
-          options: nil
+          options: nil,
+          &block
```

Comment:
The method docs should probably mention this block?

Reply:
Good suggestion. The methods that already accepted a block appear to be missing this though.

Reply:
> The methods that already accepted a block appear to be missing this though.

Good catch! That should also probably be fixed (not necessarily in this PR).